### PR TITLE
remove every language we don't care about

### PR DESCRIPTION
**Pull request by evie404**
I got sick of trying to make all the languages we don't actually use work... so I deleted everything except for closure, go, and gogo (maybe we might experiment with it one day!?).

This should make the footprint a lot smaller and maintenance a lot easier in the future.

Should we ever care for using one of those languages for protobuf, we shouldn't use rules_protobuf anyways in favor of whatever is the native Bazel rules.

build passed in: https://travis-ci.org/vsco/rules_protobuf/jobs/587324241

### CODE REVIEW

**Comment by evie404** [comment]
this actually ends up making our upgrade story significantly easier. i already have a branch on Bazel 0.26.x

File: WORKSPACE [hunk 1]
```diff
@@ -108,41 +60,10 @@ load("//protobuf:rules.bzl", "proto_repositories")
...
-
-objc_proto_repositories()
-
-load("//python:rules.bzl", "py_proto_repositories")
-
-py_proto_repositories()
-
-load("//ruby:rules.bzl", "ruby_proto_repositories")
-
-ruby_proto_repositories()
```
Comment (akakitani) [inline WORKSPACE:148]: 😢

**Review by akakitani — APPROVED** [review]
👍  just curious, do you know what the blockers have been for us using `go_proto_library` from `rules_go`?

**Comment by evie404** [comment]
@akakitani iirc, `rules_go` uses `protoc-gen-go` from golang/protobuf whereas `rules_prorobuf` uses `protoc` from protobuf. the latter has better plugin support which we need (or rather, i haven't been able to disentangle successfully)

relevant issue: https://github.com/golang/protobuf/issues/547

File: WORKSPACE (hunk 1 above)
Comment (evie404) [reply WORKSPACE:148]: i know :'(

**Review by promiseofcake — APPROVED** [review]
Wonderful, thanks!

**Pull request merged**